soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -0,0 +1,1943 @@
+﻿/* Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CHIP_CHIPREGS_H
+#define CHIP_CHIPREGS_H
+
+#include <sys/util.h>
+
+#define EC_Register_Base_Address    0x00F00000
```

Comment:
I found it, my search wasn't working since this file's diff was hidden...

Shouldn't this value come from the device tree for the board?

Reply:
OK, I've revised it, now it can be defined by devicetree, and the timer register value become come from it's timer device tree.

Reply:
Looks good to me, thank you